src/app/components/zlux-file-tree/zlux-file-tree.component.ts

```diff
@@ -113,6 +113,7 @@ export class ZluxFileTreeComponent implements OnInit, OnDestroy {
   @Output() newFileClick: EventEmitter<any> = new EventEmitter<any>();
   @Output() copyClick: EventEmitter<any> = new EventEmitter<any>();
   @Output() deleteClick: EventEmitter<any> = new EventEmitter<any>();
+  @Output() USSRenameEvent: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
you can put it under "ussSelect" as "ussRenameEvent" if we want to maintain format

Reply:
ussSelect is used somewhere else and this is a bit more than just a selection event so I think it warrants its own emitter.